generator/src/main/java/com/scottlogic/deg/generator/walker/RuntimeDecisionTreeWalkerFactory.java

```diff
@@ -40,7 +40,7 @@ public DecisionTreeWalker getDecisionTreeWalker(Path outputPath) {
                     rowSpecMerger,
                     new ExhaustiveProducer());
             case REDUCTIVE:
-                IterationVisualiser visualiser = new NoOpIterationVisualiser();
+                IterationVisualiser visualiser = new ReductiveIterationVisualiser(outputPath);
```

Comment:
Will be reverted when `5718c83b7ba5622797ef6d6731f1106534b65195` is removed from the branch